app/ui/lib/Combobox.tsx

```diff
@@ -93,17 +93,18 @@ export const Combobox = ({
         disabled={disabled || isLoading}
       >
         {label && (
-          // TODO: FieldLabel needs a real ID
           <div className="mb-2">
             <FieldLabel
-              id="FieldLabel"
+              id={`${id}-label`}
               as="div"
               tip={tooltipText}
               optional={!required && !hideOptionalTag}
             >
               <Label>{label}</Label>
             </FieldLabel>
-            {description && <TextInputHint id="TextInputHint">{description}</TextInputHint>}
+            {description && (
+              <TextInputHint id={`${id}-help-text`}>{description}</TextInputHint>
```

Comment:
This one, however, is not being magically handled by Headless. I tried manually passing `aria-labelledby` and `aria-describedby` to the `ComboboxInput` and it doesn't seem to work.

....oh.

https://headlessui.com/react/combobox#adding-a-description

Yep, it works, though you have to wrap the entire pile of Combobox stuff in a `<Field>`.

<img width="659" alt="image" src="https://github.com/user-attachments/assets/f0a3a055-560a-443e-9516-b0e08bb3f7fe">

```diff
diff --git a/app/ui/lib/Combobox.tsx b/app/ui/lib/Combobox.tsx
index a9c8a29e..598d4103 100644
--- a/app/ui/lib/Combobox.tsx
+++ b/app/ui/lib/Combobox.tsx
@@ -11,6 +11,8 @@ import {
   ComboboxInput,
   ComboboxOption,
   ComboboxOptions,
+  Description,
+  Field,
   Combobox as HCombobox,
   Label,
 } from '@headlessui/react'
@@ -34,7 +36,6 @@ export type ComboboxBaseProps = {
   placeholder?: string
   required?: boolean
   tooltipText?: string
-  ariaLabel?: string
   hideOptionalTag?: boolean
   /**
    * Pass in `allowArbitraryValues` as `true` when the user should be able to
@@ -69,7 +70,6 @@ export const Combobox = ({
   onChange,
   onInputChange,
   allowArbitraryValues = false,
-  ariaLabel,
   hideOptionalTag,
 }: ComboboxProps) => {
   const [query, setQuery] = useState(selected || '')
@@ -83,7 +83,7 @@ export const Combobox = ({
   const zIndex = usePopoverZIndex()
   const id = useId()
   return (
-    <>
+    <Field>
       <HCombobox
         value={selected}
         // fallback to '' allows clearing field to work
@@ -103,7 +103,9 @@ export const Combobox = ({
               <Label>{label}</Label>
             </FieldLabel>
             {description && (
-              <TextInputHint id={`${id}-help-text`}>{description}</TextInputHint>
+              <Description>
+                <TextInputHint id={`${id}-help-text`}>{description}</TextInputHint>
+              </Description>
             )}
           </div>
         )}
@@ -121,7 +123,6 @@ export const Combobox = ({
           )}
         >
           <ComboboxInput
-            aria-label={ariaLabel}
             displayValue={() => (selected ? selected : query)}
             onChange={(event) => {
               setQuery(event.target.value)
@@ -184,6 +185,6 @@ export const Combobox = ({
           </ComboboxOptions>
         )}
       </HCombobox>
-    </>
+    </Field>
   )
 }
```